### ISSUE TITLE
Warnings when generating ROOT dictionaries

### ISSUE THREAD

**Issue by jmcarcell** [AIDASoft/podio#581]
When compiling, this warnings appear:
```
[113/120] Generating TestDataModel_v3Dict.cxx, TestDataModel_v3DictDict.rootmap
Warning: Unused class rule: ExampleWithArrayv2
Warning: Unused class rule: ex42::ExampleWithNamespacev2
Warning: Unused class rule: ExampleWithUserInitv2
```
It would be nice for them not to appear; we should strive for clean builds without warnings or other messages. This is not very important and I'm not annoyed enough by them to have a look (yet)

**Comment by tmadlener** [comment]
Yeah, these come from the dictionary generation for the schema evolution tests. I am not sure they are completely silencable though. I think I tried for a bit, but it wasn’t trivial so I moved on.
